recipes/libftdi/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+import shutil
+
+from conans import ConanFile, CMake, tools
+
+
+class LibFtdiConan(ConanFile):
+    name = "libftdi1"
+    description = "A library to talk to FTDI chips"
+    license = ["MIT", "BSD-3-Clause"]
+    topics = ("conan", "libftdi1")
+    homepage = "http://developer.intra2net.com/git/?p=libftdi;a=summary"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _source_subfolder = "source_subfolder"
+
+    _cmake = None
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
```

Comment:
I think libftd1 needs these.
```suggestion
        os.rename(extracted_dir, self._source_subfolder)

    def requirements(self):
        self.requires("libusb/1.0.23")
        self.requires("boost/1.73.0")
        self.requires("libconfuse/3.2.2")

    def _configure_cmake(self):
```

Reply:
I'm not sure about boost anymore, because this would make the program c++.
This should be tested.